package.json

```diff
@@ -103,7 +122,7 @@
     "node-fetch": "^2.6.7",
```

Comment:
if we move `js-levenshtein` into `devDependencies - it should get bundled into the output of msw via tsup, and avoid the need to use an esm dependency for that I believe.

Reply:
I dislike a bit that we lose the semantics of what dev/regular dependencies in package.json actually mean. Can't we achieve the same via the esbuild config? I know there's a way to force-bundle any dependencies with webpack, I can imagine there being an alternative for esbuild as well. 

My suggestion here is instead of handling this by moving regular dependencies to dev, we may consider specifying what to bundle and what not on the bundler level? What do you think about this?

Reply:
Definitely agree there, I don't recall any options to force that last time I looked at tsup, but definitely worth taking another look!